### DISCUSSION TITLE
ftp utility missing

### DISCUSSION THREAD

**Discussion by rosswakelin** [Security-Onion-Solutions/securityonion#7890] [Ideas] [answered]
It would be useful if the ftp client was included in the base build, it would make it much easier to get things like update ISOs onto the master.

**Comment by dougburks** [comment, marked as answer]
The base build should include `wget` and `curl`, both of which should be able to retrieve via FTP. Alternatively, you might consider other transfer mechanisms like `sftp`, `scp`, or `rsync`.
